commons/zenoh-core/src/macros.rs

```diff
@@ -157,7 +157,7 @@ macro_rules! to_u64 {
 }
 
 // This macro allows to spawn the right amount of threads in the
-// async_std executor
+// tokio executor
```

Comment:
The `zasync_executor_init` is specific to `async-std` and it is needed for tests to run on the CI. If we are moving to *tokio* then `zasync_executor_init` shouldn't be needed anymore. This change should be reverted and any call to `zasync_executor_init` should be removed from all the tests ported to *tokio*.